packages/cli/src/reporters/github.ts

```diff
@@ -60,21 +60,9 @@ export class GithubMdBuilder {
           `${formatDuration(result.responseTime)} `,
         ]
 
-        if (this.testSessionId && testResultId) {
-          const assets: Array<string> = []
-
-          if (result.hasFailures && result.traceFilesUrls) {
-            assets.push(`[Trace](${getTraceUrl(result.traceFilesUrls[0])})`)
-          }
-
-          if (result.hasFailures && result.videoFilesUrls) {
-            assets.push(`[Video](${result.videoFilesUrls[0]})`)
-          }
-
-          const assetsColumn: string = assets.join(' \\| ')
-
-          const linkColumn = `[Full test report](${getTestSessionUrl(this.testSessionId)}/results/${testResultId})`
-          tableRow.push(assetsColumn, linkColumn)
+        if (this.testSessionId && this.testResultIds) {
```

Comment:
`this.testResultIds` doesn't exists here. Maybe something is missing after the reporters refactor. I'll check it tomorrow.